pdns/dnsdist-lua-rules.cc

```diff
@@ -892,32 +892,76 @@ std::shared_ptr<DNSRule> makeRule(const luadnsrule_t& var)
     return std::make_shared<NetmaskGroupRule>(nmg, true);
 }
 
+static boost::uuids::uuid getRuleID(std::string& id)
```

Comment:
`get...` for something that also makes something out of thin air sounds wrong to me. Weak suggestion: `makeRuleID`